ajax/libs/roll-call/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "roll-call",
+  "filename": "roll-call.min.js",
+  "description": "Roll Call is a completely free🎉 voice chat service with podcast quality recording.",
+  "author": "Mikeal Rogers <mikeal.rogers@gmail.com> (http://www.mikealrogers.com)",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/mikeal/roll-call#readme",
```

Comment:
Let's remove this homepage or replace the url with its GitHub page(if it has one, not the same as the repository), or other specified homepage, as the repository is already the default homepage, thanks!

Reply:
@anderson-sousa this PR is really good!
Let's use https://rollcall.audio as the homepage
Thanks.